src/resolvers.ts

```diff
@@ -28,7 +28,8 @@ export const resolveContentPaths = (srcDir: string, nuxt = useNuxt()) => {
   const extensionFormat = (s: string[]) => s.length > 1 ? `.{${s.join(',')}}` : `.${s.join('') || 'vue'}`
 
   const defaultExtensions = extensionFormat(['js', 'ts', 'mjs'])
-  const sfcExtensions = extensionFormat(nuxt.options.extensions.map(e => e.replace(/^\.*/, '')))
+  const extensions = Array.from(new Set(['vue', ...nuxt.options.extensions]))
+  const sfcExtensions = extensionFormat(extensions.map(e => e.replace(/^\.*/, '')))
```

Comment:
Personally I tend not to assign a variable for single-use unless it is essential for readability.

```suggestion
  const sfcExtensions = extensionFormat(Array.from(new Set(['vue', ...nuxt.options.extensions])).map(e => e.replace(/^\.*/, '')))
```